packages/backend/migrations/20230201133042_create_ledger_accounts_table.js

```diff
@@ -5,7 +5,7 @@
 exports.up = function (knex) {
   return knex.schema.createTable('ledgerAccounts', function (table) {
     table.uuid('id').notNullable().primary()
-    table.uuid('accountRef').notNullable().unique().index()
+    table.uuid('accountRef').notNullable().index()
```

Comment:
This won't be unique because Assets will use the same `accountRef` (will equal `asset.id`), but a different `type` (`LIQUIDITY_ASSET` or `SETTLEMENT`)